src/main/java/com/google/graphgeckos/dashboard/fetchers/GitHubFetcher.java

```diff
@@ -0,0 +1,23 @@
+package com.google.graphgeckos.dashboard.fetchers;
+
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class GitHubFetcher {
+
+  /**
+   * Handles POST requests from the GitHub API. Setting GitHub Webhooks is required.
+   * Extracts required data from the json {@see CommitData} and adds it to the Storage.
+   *
+   * Class that provide access to the Storage is required to implement the functionality.
+   * @param commitData json from the GitHub API
+   */
+  @PostMapping("/github-info")
+  public void postGithubInfo(@RequestBody CommitData commitData) {
```

Comment:
Apply the same change here
```suggestion
  public void postGitHubInfo(@RequestBody CommitData commitData) {
```

Reply:
Done!